### ISSUE TITLE
Do all Parameters need a min and max?

### ISSUE THREAD

**Issue by fdumont7** [FarynD/SWMM2PEST#4]
As far as I'm aware, only some parameters need a min and max. it would probably be easier to just give that option for the parameters that need it.

and if I do that, what parameters need the min and max?

**Comment by michelleannesimon** [comment]
No, not all parameters do.  Manning's Coefficient, storage depth's, and other parameters are far more likely to be adjusted than others.
